src/main/java/ch/admin/bar/siardsuite/component/rendering/FormRendererPresenter.java

```diff
@@ -0,0 +1,69 @@
+package ch.admin.bar.siardsuite.component.rendering;
+
+import ch.admin.bar.siardsuite.Controller;
+import ch.admin.bar.siardsuite.model.TreeAttributeWrapper;
+import ch.admin.bar.siardsuite.presenter.archive.browser.ChangeableDataPresenter;
+import ch.admin.bar.siardsuite.presenter.archive.browser.SearchableTableContainer;
+import ch.admin.bar.siardsuite.presenter.tree.DetailsPresenter;
+import ch.admin.bar.siardsuite.view.RootStage;
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.fxml.FXML;
+import javafx.scene.layout.VBox;
+
+public class FormRendererPresenter extends DetailsPresenter implements ChangeableDataPresenter, SearchableTableContainer {
+
+    @FXML
+    private VBox container;
+
+    private FormRenderer<?> formRenderer;
+
+    private final BooleanProperty hasChanged = new SimpleBooleanProperty(false);
+    private final BooleanProperty hasSearchableData = new SimpleBooleanProperty(false);
+
+    @Override
+    public void init(Controller controller, RootStage stage, TreeAttributeWrapper wrapper) {
+
+        this.formRenderer = new FormRenderer<>(wrapper.getRenderableForm()
+                .orElseThrow(() -> new IllegalArgumentException("No renderable form provided")),
+                hasChanged,
+                hasSearchableData);
+
+        this.container.getChildren().addAll(formRenderer.renderForm());
+    }
+
+    @Override
+    protected void bindLabels() {
+        // TODO re-render table (switch of language)
```

Comment:
Labels for forms and table column are not updated if I change the application language, as shown in the image below.
![image](https://github.com/sfa-siard/siard-suite/assets/4577216/449ed27d-b820-416a-9b75-dbe206566846)

Maybe this is related to this open TODO?

Reply:
Done